### DISCUSSION TITLE
Website Initiative: Epics & Stories

### DISCUSSION THREAD

**Discussion by jamesxv7** [Code4PuertoRico/cfpr-website#1] [General]
A través de las diferentes discusiones que han surgido tanto en Slack, [aquí](https://github.com/Code4PuertoRico/code4puertorico.github.io/discussions/) y como resultado del pasado [Hack Night](https://github.com/Code4PuertoRico/code4puertorico.github.io/discussions/1), entiendo hay material suficiente para comenzar a definir las diferentes tareas que se necesitan para trabajar el Website de Code for PR. 

Voy a recomendar utilizar ciertos elementos de la metodología Agile disponible [en la guía de Atlassian](https://www.atlassian.com/agile/project-management/user-stories), ya que entiendo proveen un buen framework para definir, asignar y “trackear” el progreso del proyecto. 

Hago la salvedad que nada de esto está escrito en piedra y que cualquier otra sugerencia es bienvenida. 

Esta discusión gira alrededor de la iniciativa del Website Oficial para Code for PR. Tenido la **iniciativa** definida, podemos ir describiendo los diferentes Epics y Stories. Para facilitar el diálogo y el desarrollo de los epics y stories voy a sugerir los siguientes perfiles o **Personas** que pueden ser de interés a nuestra iniciativa.

- Developer
- Core Member or Administrator
- UX & UI Designer
- Member of the Press
- ONG Member
- Visitor
- Government Personnel

@rubenvarela había mencionado algunas de ellas en un comentario en una [discusión ](https://github.com/Code4PuertoRico/code4puertorico.github.io/discussions/1#discussioncomment-423764)previa. 

Una vez tengamos un acuerdo de las personas podemos ir al detalle de cada epic y sus stories. 

cc @jpadilla & @froi

**Comment by froi** [comment]
Parte de esta iniciativa debe de incluir como la brigada obtiene feedback de personas que no estan comodas con utilizar GitHub. 

El repo de [Code4PR tiene un issue referente a esto](https://github.com/Code4PuertoRico/Code4PR/issues/46). Creemos que se debe de incluir en esta discusión.

CC/ @jpadilla

**Reply by rubenvarela** [reply]
@froi Referente al proceso de ahora para hacer el portal o en el futuro con el portal que se esta haciendo?

Si es ahora, podríamos por ahora hacer forms en Google Forms, añadirlos al website y darle acceso a varias personas para que se puedan crear issues con la información recibida? 

Hablo en plural ya que no se si hacen falta multiple forms/templates. 

Si es para el portal nuevo, claro, debemos tener algo más integrado.

**Comment by froi** [comment]
Tambien tenemos el tema de el codigo de conducta. @jamesxv7 trajo este tema anteriormente [aqui](https://github.com/Code4PuertoRico/Code4PR/issues/55).
